source/code-snippets/usage-examples/deleteMany.ts

```diff
@@ -1,22 +1,34 @@
+/* Delete multiple documents */
+
+// Import the MongoClient type from the mongodb package.
 import { MongoClient } from "mongodb";
 
 // Replace the uri string with your MongoDB deployment's connection string.
 const uri = "<connection string uri>";
 
+// Create a new client and connect to MongoDB.
 const client = new MongoClient(uri);
 
+// Define the Movie interface to model the data.
 interface Movie {
   title: string;
 }
 
 async function run() {
   try {
+    // Access the movies collection from the sample_mflix database.
     const database = client.db("sample_mflix");
     const movies = database.collection<Movie>("movies");
+
+    // Delete all documents where the title contains the string "Santa".
```

Comment:
Suggestion:
Instead of "where" (location), I think it could be good to use "in which" or to describe the matching.

While I think it can be helpful to include values, since this is a regular expression, it might get awkward to try to reproduce this language for a more complex regular expression. I think it could be more general since the value is apparent and self-explanatory in the code.

I think it could be helpful to skip the comment "Access the movies collection..." and reference the collection in this comment instead since this is closer to the usage of the collection. E.g.

"Delete all documents that match the specified regular expression in the title field from the "movies" collection."

Reply:
I think this is also inconsistent with the other comment in the JS file even though the code is the same:
"// Delete all documents that match the preceding query filter."